redis.js

```diff
@@ -0,0 +1,20 @@
+import Redis from 'ioredis'
+
+const redis = process.env.TENANT_SEED_TESTING
+  ? null
+  : new Redis(process.env.REDIS_URI ?? 'localhost:6379')
+
+export async function allTenants() {
+  if (process.env.TENANT_SEED_TESTING) return []
+  const keys = await redis.keys('tenantPrivate:*')
```

Comment:
https://redis.io/docs/latest/commands/keys// There are performance warnings for using `keys()` instead of `scan({match: 'tenantPrivate:*'})`.

The ioredis docs provide an [example](https://github.com/redis/ioredis?tab=readme-ov-file#streamify-scanning).

Perhaps consider using scan, which chunks the scanning so that concurrent requests aren't blocked for as long. Though apparently it would take our redis having quite enormous scale to have much of a consequence. And this operation runs rarely.